packages/Datadog.Unity/Tests/Rum/DatadogTrackedWebRequestTest.cs

```diff
@@ -135,8 +221,19 @@ private static void VerifyHeaders(Dictionary<string, string> headers, TracingHea
                     Assert.AreEqual("rum", headers["x-datadog-origin"]);
                     var traceString = headers.GetValueOrDefault("x-datadog-trace-id");
                     var spanString = headers.GetValueOrDefault("x-datadog-parent-id");
+                    var tagsHeader = headers.GetValueOrDefault("x-datadog-tags");
+                    var tagParts = tagsHeader?.Split("=");
                     if (sampled)
                     {
+                        Assert.IsNotNull(tagParts);
+                        if (tagParts != null)
+                        {
+                            Assert.AreEqual(tagParts[0], "_dd.p.tid");
+                            BigInteger.TryParse(tagParts[1], NumberStyles.HexNumber, null, out var highTraceInt);
```

Comment:
Yeah same as above, but parsable to `ulong` not `int`

Reply:
Actually, because this test is checking that the result fits in 63 bits, the use of `BigInteger.GetByteCount` simplifies it, so I'll probably leave it as is.